lib/cloudstore/rate_limiter.es6.js

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
The rateLimiter and rateLimiterDAO will only be used with datastore. I feel it's fine putting these files in cloud datastore directory.